packages/types/src/interfaces/augment-types.ts

```diff
@@ -1815,6 +1815,12 @@ declare module '@polkadot/types/types/registry' {
     RewardBalanceOf: RewardBalanceOf;
     'Option<RewardBalanceOf>': Option<RewardBalanceOf>;
     'Vec<RewardBalanceOf>': Vec<RewardBalanceOf>;
+    // @ts-ignore
+    RewardDestination: RewardDestination;
+    // @ts-ignore
```

Comment:
had to add ts-ignore as RewardDestination already exist in polkadot.. and so, it would give - TS2717

Subsequent property declarations must have the same type.  Property 'RewardDestination' must be of type 'RewardDestination', but here has type 'RewardDestination'.

Reply:
please add a comment to the file about this

Reply:
done